composer.json

```
@@ -9,8 +9,10 @@
   "minimum-stability": "dev",
   "require": {
     "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
-    "pimcore/pimcore": "^11.0",
-    "pimcore/output-data-config-toolkit-bundle": "^4.1 || ^5.0"
+    "pimcore/admin-ui-classic-bundle": "^1.6",
```

```suggestion
    "pimcore/admin-ui-classic-bundle": "^1.0",
```
i have just realized that we do not have a strict hard requirement for latest admin ui `1.6`, should be possible to use it with a lower one

---

but does it hurt to also update to the latest admin if we have the chance?

---

It doesn't hurt and i always prefer to have the latest but having in mind cases like LTS, platform version etc.. it could be kind of unnecessary(without a specific reason) limiting in some way to require the 1.6 as minimum.

Imho letting composer do the magic to get the one that fits the combination would be better in this case/bundle